pyaedt/emit.py

```diff
@@ -247,6 +247,20 @@ def set_max_simultaneous_interferers(self, val):
         eng = self.emit_obj._emit_api.get_engine()
         eng.max_simultaneous_interferers = val
 
+    @pyaedt_function_handler()
+    def is_domain_valid(self, ret_val, domain):
+
+        """
+        Return true if the given domain is valid for the current Revision
+
+        Examples
+        ----------
+        >>> domain = aedtapp.interaction_domain()
+        >>> if aedtapp.results.is_domain_valid(domain):
```

Comment:
This way we can also see the returned value.
```suggestion
        >>> aedtapp.results.is_domain_valid(domain)
        True
```